modular_skyrat/modules/customization/game/objects/items/plushes.dm

```diff
@@ -131,6 +134,7 @@
 	attack_verb_continuous = list("baps")
 	attack_verb_simple = list("bap")
 	squeak_override = list('modular_skyrat/modules/emotes/sound/voice/weh.ogg' = 1)
+	gender = FEMALE
```

Comment:
wtf

Reply:
Sarmie a male bro

Reply:
doubt

Reply:
The first person to say so lied, I dont know everyone.